DataFormats/Histograms/interface/MonitorElementCollection.h

```diff
@@ -138,6 +138,7 @@ struct MonitorElementData {
     TH2S = 0x21,
     TH2D = 0x22,
     TH2I = 0x23,
+    TH2Poly = 0x24,
```

Comment:
I'd prefer to incorporate the `TH2Poly` via an existing PR https://github.com/cms-sw/cmssw/pull/41932 instead of part of a many-thousand-line PR.

Reply:
FTR: This was removed from this PR.